ld/stub.x

```diff
@@ -44,13 +44,13 @@ REGION_ALIAS("REGION_RWTEXT", IRAM);
 REGION_ALIAS("REGION_RTC_FAST", RTC_FAST);
 
 
-ENTRY(_start_hal) /* _start_hal */
+ENTRY(_start_hal)
 PROVIDE(_start_trap = _start_trap_hal);
 
 PROVIDE(_stext = ORIGIN(REGION_TEXT));
 PROVIDE(_stack_start = ORIGIN(REGION_STACK) + LENGTH(REGION_STACK));
 PROVIDE(_max_hart_id = 0);
-PROVIDE(_hart_stack_size = 32K);
+PROVIDE(_hart_stack_size = 2K);
```

Comment:
Why reduce the stack size? The stub doesn't use much RAM and having a large stack won't hurt in any way.

Reply:
NVM i see you fixed it.